options.json

```diff
@@ -1,4 +1,6 @@
 {
+  "compatibilityMode": false,
+  "copyCodeButton": true,
```

Comment:
Personally I'd shorten this option's name to "copyButtons" or something! But also, like, whatever Rafe.......

Reply:
Will do that